service/src/main/java/uk/gov/hmcts/reform/fpl/model/PartyApplicant.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.hmcts.reform.fpl.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import uk.gov.hmcts.reform.fpl.model.common.EmailAddress;
+import uk.gov.hmcts.reform.fpl.model.common.Party;
+import uk.gov.hmcts.reform.fpl.model.common.TelephoneNumber;
+
+import java.util.Date;
+
+@Data
+@EqualsAndHashCode(callSuper = false)
+public class PartyApplicant extends Party {
+    private final String name;
+    private final TelephoneNumber mobileNumber;
+    private final String jobTitle;
+    @JsonProperty("pbaNumber")
```

Comment:
@JsonProperty is not necessary as the variable is named the same

Reply:
👍